documentation/topics/attributes.md

```diff
@@ -0,0 +1,105 @@
+# Attributes
+
+Attributes specify the name, type and options of a piece of information in a resource.
+
+## Ways of writing attributes
+
+There are two ways to write an attribute:
+
+```elixir
+defmodule MyProject.MyApi.MyResource do
+  use Ash.Resource, data_layer: ...
+
+  actions do
+
+  end
+
+  attributes do
+    attribute :name, :string, allow_nil?: false
+
+    # or ...
+    attribute :name, :string do
+      allow_nil? false
+    end
+  end
+end
+```
+
+Both ways will work. Though when you're using many options the latter is preferred.
+
+For more information on attribute types including composite types and defining your own custom type see t:Ash.Type
```

Comment:
the way to link to a module is to just put the module name in backticks, i.e `Ash.Type`.

Reply:
addressed [bb4612c](https://github.com/ash-project/ash/pull/514/commits/bb4612c8519ebd737044fba86b3fd6ebf92ee9a3)